Bottom: dc6194b380715745e4b98f09d88834e4798ea3da
Top:    dae62562f0260cea80405e57cb55f68d4785db15
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-23 12:12:27 -0400

Refresh of mpc5200-dma

---

diff --git a/sound/soc/fsl/Kconfig b/sound/soc/fsl/Kconfig
index dc79bdf..1918c78 100644
--- a/sound/soc/fsl/Kconfig
+++ b/sound/soc/fsl/Kconfig
@@ -25,7 +25,6 @@ config SND_SOC_MPC8610_HPCD
 config SND_SOC_MPC5200_I2S
 	tristate "Freescale MPC5200 PSC in I2S mode driver"
 	depends on PPC_MPC52xx && PPC_BESTCOMM
-	select SND_SOC_OF_SIMPLE
 	select SND_MPC52xx_DMA
 	select PPC_BESTCOMM_GEN_BD
 	help
